Apromore-Custom-Plugins/SplitMiner-Logic/src/main/java/org/apromore/splitminer/SplitMiner.java

```diff
@@ -346,13 +346,16 @@ private void transformDFGPintoBPMN() {
 
         if(removeLoopActivities) helper.removeLoopActivityMarkers(bpmnDiagram);
 
+        // Bruce 30.04.2021: comment the below to produce diagrams without mixed split-join gateways
+        /*
         if( replaceIORs ) {
             helper.collapseSplitGateways(bpmnDiagram);
             helper.collapseJoinGateways(bpmnDiagram);
         } else {
             helper.collapseSplitGateways(bpmnDiagram);
             helper.collapseJoinGateways(bpmnDiagram);
         }
+        */
```

Comment:
Glancing over the rest of the code, you seem to have put some effort into making `replaceIORs` a configurable option.  Do you have a reason not to do it as
```
if( replaceIORs ) {
    helper.collapseSplitGateways(bpmnDiagram);
    helper.collapseJoinGateways(bpmnDiagram);
}
```
and pass `replaceIORs` as false?